data/wanderer jobs.txt

```diff
@@ -0,0 +1,605 @@
+	to offer
+		has "language: Wanderer"
+		random < 60
+	source
+		government "Wanderer"
+	destination
+		distance 2 8
+		government "Wanderer"
+	on complete
+		payment
+		payment 20000
+		dialog "You wish the biologists the best of luck on <planet> as they carry their scientific equipment off of your ship and hand you your payment."
+
+mission "Wanderer Botanists [0]"
+	name "New plant found on <planet>"
+	job
+	repeat
+	description `These <bunks> Wanderer botanists wish to get to <destination> with their <cargo> to study a recently discovered plant species. Payment will be <payment>.`
+	passengers 4 4 .2
+	cargo "scientific equipment" 5 2 .1
+	to offer
+		has "language: Wanderer"
+		random < 70
+	source
+		government "Wanderer"
+	destination
+		distance 2 8
+		government "Wanderer"
+	on complete
+		payment
+		payment 15000
+		dialog "You wish the botanists the best of luck on <planet> as they carry their scientific equipment off of your ship and hand you your payment."
+
+mission "Wanderer Botanists [1]"
+	name "New plant found on <planet>"
+	job
+	repeat
+	description `These <bunks> Wanderer botanists wish to get to <destination> with their <cargo> to study a recently discovered plant species. Payment will be <payment>.`
+	passengers 4 4 .1
+	cargo "scientific equipment" 5 2 .1
+	to offer
+		has "language: Wanderer"
+		random < 60
+	source
+		government "Wanderer"
+	destination
+		distance 2 8
+		government "Wanderer"
+	on complete
+		payment
+		payment 20000
+		dialog "You wish the botanists the best of luck on <planet> as they carry their scientific equipment off of your ship and hand you your payment."
+
+mission "Wanderer Harvest [0]"
+	name "Food harvest on <planet>"
+	job
+	repeat
+	description `Bring these <bunks> Wanderers to <destination> to assist in the food harvest. Payment is <payment>.`
+	passengers 2 10 .3
+	to offer
+		has "language: Wanderer"
+		random < 60
+	source
+		government "Wanderer"
+	destination
+		distance 2 8
+		attributes farming
+		government "Wanderer"
+	on complete
+		payment
+		payment 15000
+		dialog "You wish the Wanderers the best of luck on <planet> and collect your payment."
+
+mission "Wanderer Harvest [1]"
+	name "Food harvest on <planet>"
+	job
+	repeat
+	description `Bring these <bunks> Wanderers to <destination> to assist in the food harvest. Payment is <payment>.`
+	passengers 2 10 .2
+	to offer
+		has "language: Wanderer"
+		random < 50
+	source
+		government "Wanderer"
+	destination
+		distance 2 8
+		attributes farming
+		government "Wanderer"
+	on complete
+		payment
+		payment 20000
+		dialog "You wish the Wanderers the best of luck on <planet> and collect your payment."
+
+mission "Wanderer Invasive Species [0]"
+	name "Invasive species on <planet>"
+	job
+	repeat
+	description `These <bunks> Wanderers need to get to <stopovers> with their <cargo> by <date> to contain an invasive animal species. Return the Wanderers here with the animals for a payment of <payment>.`
```

Comment:
Return the Wanderers to <origin> with the animals, instead? You can view the description anywhere, so here doesn't really tell you where.